src/server/services/duplicatesService.js

```diff
@@ -0,0 +1,33 @@
+const fs = require('fs');
+
+const getDuplicates = require('./helpers/getDuplicates');
+
+const inputFilePath = "src/inputData/";
+const outputFilePath = "src/outputData/";
+
+const indexFilePath = "./src/dataBase/file-indexData.json";
+
+module.exports = function() {
+    const getFileIndex = require('./helpers/readFileIndex');
```

Comment:
move above

Reply:
I have trouble with these constants, I want them to get new information every time I call them, do I something wrong?